### ISSUE TITLE
Vuls in server not outputing the same JSON as local scan

### ISSUE THREAD

**Issue by dstephens1980** [future-architect/vuls#1054]
# What did you do? (required. The issue will be **closed** when not provided.)
Installed Vuls manually on Centos 7 VM with VulsRepo.  Scanned local system to establish results directory and verified the local scan results showed in VulsRepo.  Started Vuls in server mode and executed the required CURL command to send scan from client to Vuls.  Verified Vuls created JSON for client in results directory.  Verfied VulsRepo can see the client JSON.  Upon loading the client JSON, VulsRepo sits at "Please Wait" with the spinning wheel.

# Troubleshooting
I checked the HEAD of both the local scan and the remote client scan and there is a discrepency in the reported JSON version, not sure if this is cause of error or not.

Local Scan (which shows fine in VulsRepo):
{"jsonVersion":4

Remote Scan (fails to load):
{"jsonVersion":0

# What did you expect to happen?
VulsRepo should load the results of the client scan.

# What happened instead?
VulsRepo sits at "Please Wait"

* Current Output
see above

Please re-run the command using ```-debug``` and provide the output below.

# Steps to reproduce the behaviour
Everything was outlined above

# Configuration (**MUST** fill this out):

* Go version (`go version`):
go version go1.15.2 linux/amd64

* Go environment (`go env`):
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/vuls/.cache/go-build"
GOENV="/home/vuls/.config/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOINSECURE=""
GOMODCACHE="/opt/vuls/go/pkg/mod"
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/opt/vuls/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/local/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
GCCGO="gccgo"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build255068179=/tmp/go-build -gno-record-gcc-switches"

* Vuls environment:

Hash : vuls v0.12.3 build-20200929_034814_4b680b9

* config.toml:
[servers]

[servers.localhost]
host = "localhost"
port = "local"

* command:
vuls server -listen 0.0.0.0:5111 -to-localfile -format-json

**Comment by Jiab77** [comment]
@dstephens1980 I've seen the same issue but not with the same impact while running the same command as you do.

### Description:

When running a local scan, both `warnings` and `errors` object properties are in type `array` but when running a remote scan (client to server), both properties gets the `null` type instead of `array`.

This breaks the report display in [vulsrepo](https://github.com/ishiDACo/vulsrepo). I've already open an issue to propose a patch that could handle both types here: https://github.com/ishiDACo/vulsrepo/issues/165

### Observed:

__Client to Server:__

```bash
cat remote-scan.json | grep -A1 '"errors"'
    "errors": null,
    "warnings": null,
```

__Local scan:__

```bash
cat localhost.json | grep -A1 '"errors"'
    "errors": [],
    "warnings": [],
```

__Current code in [vulsrepo](https://github.com/ishiDACo/vulsrepo)__

![image](https://user-images.githubusercontent.com/9881407/99079045-ac063000-25bf-11eb-9b0c-bf717eff6bc0.png)

> The current code is not able to handle the `null` type.

### Temporary workaround:

```bash
find ${REPORT_FOLDER} -type f -iname "*.json" \
-exec echo "Patching file {} ..." \; \
-exec cp -v "{}" "{}.bak" \; \
-exec sed -e 's/"errors"\: null/"errors"\: \[\]/' -i "{}" \; \
-exec sed -e 's/"warnings"\: null/"warnings"\: \[\]/' -i "{}" \;
```

> Just replace `${REPORT_FOLDER}` by the real location of the `results` folder.

**Comment by ishiDACo** [comment]
Fixed in VulsRepo v0.2.5.
Please use this one.
